src/packages/calendar/calendarviewmodeitem.taro.tsx

```
@@ -0,0 +1,843 @@
+          if (compareDate(propEndDate, currentDate[1])) {
+            currentDate[1] = propEndDate
+          }
+          defaultData = [
+            ...splitDate(currentDate[0]),
+            ...splitDate(currentDate[1]),
+          ]
+          break
+        case 'multiple':
+          // eslint-disable-next-line no-case-declarations
+          const defaultArr = [] as string[]
+          // eslint-disable-next-line no-case-declarations
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)